src/platform/packages/shared/content-management/favorites/favorites_server/src/favorites_usage_collection.ts

```diff
@@ -115,11 +115,12 @@ export function registerFavoritesUsageCollection({
           []) as estypes.AggregationsStringTermsBucket[];
 
         typesBuckets.forEach((bucket) => {
-          favoritesUsage[bucket.key] = {
-            total: bucket.stats.sum,
-            total_users_spaces: bucket.stats.count,
-            avg_per_user_per_space: bucket.stats.avg,
-            max_per_user_per_space: bucket.stats.max,
+          const stats = bucket.stats as estypes.AggregationsStatsAggregate;
+          favoritesUsage[`${bucket.key}`] = {
```

Comment:
nit: I think this line should not have been changed
```suggestion
          favoritesUsage[bucket.key] = {
```

Reply:
I get these errors.
<img width="628" alt="image" src="https://github.com/user-attachments/assets/3d43d5a0-2ab6-4ab3-bfe9-26417429c899" />

It's caused by the new definition of `FieldValue`, which is now defined as:

```ts
export type FieldValue = long | double | string | boolean | null;
```

https://github.com/elastic/elasticsearch-js/blob/main/src/api/types.ts#L2937